packages/nextjs/app/builders/0x6d3321b4637D8451740Cf400848602f838F7A594/page.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { FaGithub } from "react-icons/fa";
+import { SiGmail } from "react-icons/si";
+import { Address } from "~~/components/scaffold-eth";
+import { BlockieAvatar } from "~~/components/scaffold-eth";
+
+const PersonalPage = () => {
+  return (
+    <div className=" flex justify-center items-center">
+      <div>
+        <div className=" pt-24 lg:pt-[100px]">
+          <div className="flex flex-col-reverse lg:flex-row items-center justify-center gap-x-[50px] md:mx-[550px]">
```

Comment:
Please double-check the mobile responsiveness, as it appears differently on smaller screens, as shown in the picture.

<img width="795" alt="Screenshot 2024-09-26 at 18 28 34" src="https://github.com/user-attachments/assets/29e4f555-132c-4f4c-a25d-c9498e1dd2ef">